test/new-e2e/system-probe/system-probe-test-env.go

```diff
@@ -0,0 +1,136 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package systemProbe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/utils/credentials"
+	"github.com/DataDog/datadog-agent/test/new-e2e/utils/infra"
+	"github.com/DataDog/test-infra-definitions/aws"
+	"github.com/DataDog/test-infra-definitions/aws/scenarios/microVMs/microvms"
+	"github.com/DataDog/test-infra-definitions/command"
+
+	"github.com/pulumi/pulumi-command/sdk/go/command/remote"
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+type TestEnv struct {
+	context context.Context
+	envName string
+	name    string
+
+	ARM64InstanceIP  string
+	X86_64InstanceIP string
+	StackOutput      auto.UpResult
+}
+
+var (
+	SSHKeyFile           = filepath.Join(".", "/", "aws-ssh-key")
+	vmConfig             = filepath.Join(".", "system-probe", "config", "vmconfig.json")
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can use go `embed` to read this file, advantage is it fails building if the file does not exist